sqlbrite/src/androidTest/java/com/squareup/sqlbrite/BriteDatabaseTest.java

```diff
@@ -207,30 +207,30 @@
     }
   }
 
-  @Test public void queryMapToOneOrNull() {
+  @Test public void queryMapToOneOrDefault() {
     Employee employees = db.createQuery(TABLE_EMPLOYEE, SELECT_EMPLOYEES + " LIMIT 1")
-        .mapToOneOrNull(Employee.MAPPER)
+        .mapToOneOrDefault(Employee.MAPPER, null)
         .toBlocking()
         .first();
     assertThat(employees).isEqualTo(new Employee("alice", "Alice Allison"));
   }
 
-  @Test public void queryMapToOneOrNullEmpty() {
+  @Test public void queryMapToOneOrDefaultEmpty() {
     Employee employees = db.createQuery(TABLE_EMPLOYEE, SELECT_EMPLOYEES + " WHERE 1=2")
-        .mapToOneOrNull(Employee.MAPPER)
+        .mapToOneOrDefault(Employee.MAPPER, null)
         .toBlocking()
         .first();
     assertThat(employees).isNull();
```

Comment:
Can you duplicate this test and use a non-null default? Maybe rename this one to "EmptyUsingNull" or something.